code/modules/xenomorph/xeno_structures.dm

```diff
@@ -600,6 +600,16 @@ TUNNEL
 
 	var/datum/effect_system/smoke_spread/xeno/acid/acid_smoke
 
+	if(charges >= 2)
+		for(var/obj/item/explosive/grenade/sticky/sticky_bomb in stepper.contents)
+			sticky_bomb.clean_refs()
+			qdel(sticky_bomb)
+			if(isxeno(stepper))
+				charges-= 2
+				acid_smoke = new(get_turf(stepper))
+				acid_smoke.set_up(1, src)
+				acid_smoke.start()
+				update_icon()
```

Comment:
Dont get this isxeno check, why is it here?
```suggestion
			if(isxeno(stepper))
				charges -= 2
				acid_smoke = new(get_turf(stepper))
				acid_smoke.set_up(1, src)
				acid_smoke.start()
				update_icon()
```

Reply:
its so that when the acid well is triggered to clear stickies it leaves acid smoke like it does fire. so I copied how its done for fire as I didn't know a simpler way to do it.